test/AlgAssAbsOrd/Ideal.jl

```diff
@@ -199,4 +199,12 @@
   I = 1 * O
   @test Hecke.is_full_lattice(I)
 
+  # parent
+  let
+    A = zero_algebra(QQ)
+    O = Order(A, elem_type(A)[])
+    I = 1 * O
+    @test parent(I) == parent(I)
+    @test hash(parent(I)) == hash(parent(I))
```

Comment:
what is this supposed to test? Maybe you meant `O` instead of `I` for some of them?

Reply:
hashing for `AlgAssAbsOrdIdlSet`

Reply:
But isn't `parent(I) === parent(I)`? (and thus these tests basically useless) If not, sorry for my confusion

Reply:
Hm no, it is not:
```
julia> parent(I) == parent(I)
true

julia> parent(I) === parent(I)
false
```
I am not super happy with the design of some of these things, but I don't want to change it here.